bin/flock

```diff
@@ -44,18 +45,34 @@ if (args.find(x => x === '--help')) {
 if (args[0] === 'create') {
   const fileName = args[1]
 
-  // NOTE: We'll need to support creating migrartions for specific plugins perhaps?
+  // migration which creates a new table
+  if (args.find(x => x === '--new-table')) {
 
-  if (fileName) {
-    const r = fs.createReadStream(path.resolve(__dirname, '../templates/migration.js'))
-    const w = fs.createWriteStream(fileName)
-    r.pipe(w)
-    r.on('end', () => {
-      process.exit(0)
-    })
+    if (fileName) {
+      const blah = 'asdfghjkl'
```

Comment:
What's this? Blah! Remove this please.

Reply:
Oops! That is test code left accidentally behind.